server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/java/org/apache/james/jmap/draft/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -340,14 +340,17 @@ public void userShouldBeSubscribedOnCreatedMailboxWhenCreateChildOfInboxMailbox(
 
     @Test
     public void subscriptionUserShouldBeChangedWhenUpdateMailbox() throws Exception {
-        MailboxId mailboxId = mailboxProbe.createMailbox(MailboxConstants.USER_NAMESPACE, username.asString(), "root");
+        MailboxId parentId = mailboxProbe.createMailbox(MailboxConstants.USER_NAMESPACE, username.asString(), "root");
+
+        String initialMailboxName = "myBox";
+        String mailboxId = createSubMailBox(parentId.serialize(), initialMailboxName);
```

Comment:
Why do we need that change?

Reply:
when we create a mailbox with `mailboxProbe` it not subscribed yet then rename would dont affect the subscription of it.

Reply:
I propose the alternative of adding a `MailboxProbeImpl::subscribe` method that we can instead use here.

Reply:
Or we can create the first mailbox via JMAP ( even easier)

Reply:
> Or we can create the first mailbox via JMAP ( even easier)

i agree with it.

> I propose the alternative of adding a MailboxProbeImpl::subscribe method that we can instead use here.
 i tried but somehow it not working like i expected.

Reply:
> i tried but somehow it not working like i expected.

Why?